pgml-extension/src/bindings/transformers/mod.rs

```diff
@@ -18,11 +25,18 @@ pub mod whitelist;
 
 create_pymodule!("/src/bindings/transformers/transformers.py");
 
+lazy_static! {
+    // Record the previous applied ENVs.
+    static ref ENVS_APPLIED: RwLock<Box<BTreeMap<&'static str, String>>> =
```

Comment:
I'm struggling to understand the purpose of this. Why do we need to record what environment variables we've applied? Can't we just set them every time or they can only be set on initialization before HF has been imported?

Reply:
The purpose of the recorded map is to warn users if the GUCs are changed and the changes can not take effect in the current session. This prevents users from downloading/caching the models in the old cache path.

Reply:
Why can't we set the env using the latest GUC value every time?

Reply:
The value of the latest GUC can not be seen by the old psql session started before changing the GUC.